content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Manual rotation of CA certificates
```

Comment:
nit:
```suggestion
title: Manual Rotation of CA Certificates
```
also consider editing the title of https://kubernetes.io/docs/tasks/tls/certificate-rotation/ to, eg, “Configure Certificate Rotation for the Kubelet”